merge_baseline_configuration_xmls.py

```diff
@@ -93,7 +98,10 @@ def add_assay_groups_from_accession(path, accession, assay_groups: ET.Element, c
     >>> children[28].get("label") == f"tonsil; {accession}"
     True
     """
-    conf_xml_path = f"{path}/{accession}-configuration.xml"
+    if args.use_subdir == "True":  
```

Comment:
When you change the flag behaviour, then args.use_subdir will be a boolean and you will be able to do:
```suggestion
    if args.use_subdir:  
```